bin/git-overwritten

```diff
@@ -22,6 +22,9 @@ unset head
 unset base
 unset diff_type
 
+# Figure out what argument to use for date as GNU & BSD `date`s are different
+[[ $(date -r123 '+%s' 2>/dev/null) -eq 123 ]] && DATEARG='-r' || DATEARG='-d@'
```

Comment:
@mislav Your original version did indeed fail on GNU, but not the one I submitted. Neither of the issues you suggest are actually a concern because I used the bash keyword syntax `[[` for the test per [my previous comment](https://github.com/mislav/dotfiles/pull/12#commitcomment-12462290). They only would have been a concern with your use of `/usr/bin/[` (or bash's emulation of it) for the test. See [this post](http://unix.stackexchange.com/a/32227/1925) for background on test syntaxes is bash.